src/en/caas.md

```diff
@@ -0,0 +1,192 @@
+Title: CAAS and Juju
+TODO:  Once 2.4 is officially released remove support Note and update status output
+       Should eventually link to CAAS-charm developer documentation
+       Update when storage becomes a Juju drivable aspect.
+       Add architectural overview/diagram once CAAS becomes stable.
+       Consider manually adding a cluster (third-party installs) via `add-cloud` and `add-credential`
+
+# CAAS and Juju
+
+CAAS is *Containers as a Service*, a cloud service that allows users to upload,
+organize, run, scale, manage, and stop containers. Combining this with Juju
+opens up new practical benefits for Juju users. Currently, Juju supports
+Kubernetes as its underlying CAAS solution.
+
+Kubernetes (often abbreviated as "k8s") provides a flexible architecture for
+managing containerised applications at scale (see the
+[Kubernetes documentation][upstream-kubernetes-docs] for more information). It
+most commonly employs Docker as its container technology.
+
+## CAAS-specific workflow
+
+Here we discuss how a building and working with a CAAS environment may differ
+from a standard Juju workflow.
+
+The only CAAS-specific Juju command is `add-k8s`, which acts as the `add-cloud`
```

Comment:
add-k8s does 2 things:
- add-cloud
- add-credential
It creates a cloud definition for the k8s cluster and imports a credential and CA cert from the k8s config so Juju can talk to the cluster. The add-k8s command is currently undergoing a little refinement as per https://bugs.launchpad.net/bugs/1768837